index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odin-Landing-Page!</title>
+    <link rel="stylesheet" href="style.css" >
+  </head>
+  <body>
+    <header>
+      <div class="header-top">
+        <h1 id="logo">Header Logo</h1>
+        <nav>
+          <ul>
+            <li><a href="#">header link one</a></li>
+            <li><a href="#">header link two</a></li>
+            <li><a href="#">header link three</a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="header-bottom">
+      <div class="header-hero">
+        <h1>This Website is awesome</h1>
+        <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+        </p>
+        <button class="header-button">Sign Up</button>
+    </div>
+    <div >
+        <img class="header-img" src="img/header_img.png" alt=" " >
+    </div>
+    </div>
+</header>
+<main>
+    <section class="section-two">
+    <h2>Some Random Information</h2> 
+    <div class="info-container">
+        <div class="info-box">
+        <img src="img/sec_img1.jpg" alt=" " >
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+        <div class="info-box">
+        <img src="img/sec_img2.jpg" alt=" " >
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+        <div class="info-box">
+        <img src="img/sec_img3.jpg" alt=" " >
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+        <div class="info-box">
+        <img src="img/sec_img4.jpg" alt=" " >
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+    </div>
+    </section>
+    <section class="section-three">     
```

Comment:
- [ ] [OPTIONAL] Good job so far for using the inline comments :+1:  But please phrase the comments politely, as suggestions instead of orders "Please ..." or "I think ..." :smile: